src/schema/fair.ts

```diff
@@ -254,6 +264,7 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
     },
     start_at: date,
     end_at: date,
+    active_start_at: date,
```

Comment:
Cargo-culting [this date field definition](https://github.com/artsy/metaphysics/blob/master/src/schema/fields/date.ts) which is already used for `start_at` and `end_at`